src/main/scala/Ch3.scala

```diff
@@ -0,0 +1,115 @@
+package swscala
+
+
+object Ch3Exercises {
+
+  /**
+    Problem 1
+
+    id(id)
+    def id[T]: (T => T) = t => t
+    can be rewritten as
+    id[T](t: T): T = t
+    pass in argument id[U]: U => U
+    id[T](id[U]: U => U): T
+    substitute T with (U => U)
+    id[U => U](id[U]: U => U): U => U
+    combine id(id) to idid
```

Comment:
So, what does id(id) do? Implement `id` as written above, and write tests for `id(id)`.